test/service/test_response.py

```diff
@@ -2386,21 +2386,33 @@ def params(self,
     def test_file_source_facet(self):
         """
         Verify the 'fileSource' facet is populated with the human-readable
-        versions of the name used to generate the 'submitter_id' UUID.
+        versions of the name used to generate the 'submitter_id' UUID, and that
+        the facet values match the hits[].files[].fileSource values.
         """
         params = self.params(project_id='8185730f-4113-40d3-9cc3-929271784c2b')
         url = self.base_url.set(path='/index/files', args=params)
         response = requests.get(str(url))
         response.raise_for_status()
         response_json = response.json()
         facets = response_json['termFacets']
-        expected = [
-            {'term': 'DCP/2 Analysis', 'count': 3},
-            {'term': 'DCP/1 Matrix Service', 'count': 3},
-            {'term': 'HCA Release', 'count': 1},
-            {'term': 'ArrayExpress', 'count': 7}
-        ]
-        self.assertElasticsearchResultsEqual(expected, facets['fileSource']['terms'])
+        expected_counts = {
+            'DCP/2 Analysis': 3,
+            'DCP/1 Matrix Service': 3,
+            'HCA Release': 1,
+            'ArrayExpress': 7
+        }
+        expected_facets = [{'term': key, 'count': val}
+                           for key, val in expected_counts.items()]
+        self.assertElasticsearchResultsEqual(expected_facets,
+                                             facets['fileSource']['terms'])
+        actual_counts = Counter()
+        for hit in response_json['hits']:
+            file = one(hit['files'])
+            # FIXME: Remove deprecated field 'source'
+            #        https://github.com/DataBiosphere/azul/issues/3180
+            self.assertEqual(file['source'], file['fileSource'], file['source'])
```

Comment:
Third argument is redundant as the `assert.*Equal` methods always show their non-matching arguments. 
```suggestion
            self.assertEqual(file['source'], file['fileSource'])
```